src/main/java/sirius/biz/tenants/mongo/MongoUserAccountImportHandler.java

```diff
@@ -61,29 +72,78 @@ protected MongoUserAccountImportHandler(Class<?> clazz, ImporterContext context)
     }
 
     @Override
-    public Optional<MongoUserAccount> tryFind(Context data) {
-        if (data.containsKey(MongoUserAccount.ID.getName())) {
+    protected MongoUserAccount loadForFind(Context data) {
+        MongoUserAccount userAccount = load(data,
+                                            MongoUserAccount.ID,
+                                            MongoUserAccount.TENANT,
+                                            SQLUserAccount.USER_ACCOUNT_DATA.inner(UserAccountData.EMAIL),
+                                            MongoUserAccount.USER_ACCOUNT_DATA.inner(UserAccountData.LOGIN)
+                                                                              .inner(LoginData.USERNAME));
+
+        if (UserContext.getCurrentUser().hasPermission(TenantUserManager.PERMISSION_SYSTEM_TENANT_MEMBER)) {
```

Comment:
why there is no such permission check for SQLUserAccountImportHandler ?

Reply:
Is there already: https://github.com/scireum/sirius-biz/blob/5bb3e5ceb1ecfa21987907d4e39c9a8aa1ef6231/src/main/java/sirius/biz/tenants/jdbc/SQLUserAccountImportHandler.java#L121